miro_preprocessor/image_sorter/src/test_miro_image_sorter.py

```diff
@@ -76,11 +76,17 @@ def _get_msg(sqs_client, queue_url):
     )
 
     message_body = messages['Messages'][0]['Body']
-    inner_message = json.loads(message_body)['Message']
+    json_decoded_message = json.loads(message_body)
 
-    return json.loads(
-        json.loads(inner_message)['default']
-    )
+    inner_message = json_decoded_message['Message']
+    subject = json_decoded_message['Subject']
+
+    return {
+        "subject": subject,
+        "message": json.loads(
+            json.loads(inner_message)['default']
+        )
```

Comment:
Why modify the return structure if you never assert on anything but the subject?

Reply:
I want to, there's a but there's a bug in moto which hard codes the value.